addons/tls13-middlebox/install.rdf

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RDF xmlns="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:em="http://www.mozilla.org/2004/em-rdf#">
+  <Description about="urn:mozilla:install-manifest">
+    <em:id>tls13-middlebox@mozilla.org</em:id>
+    <em:version>1.0.0</em:version>
+    <em:type>128</em:type>
```

Comment:
As noted in the bug this should be type 2 - 128 is the type for telemetry experiments which skips the compat checking in install.rdf and it works via about:debugging which is why I didn't catch this earlier sorry :/